utils/firebaseNotification.js

```diff
@@ -0,0 +1,74 @@
+import messaging from '@react-native-firebase/messaging'
+import { Alert } from 'react-native';
+
+
+export default class FirebaseNotification{
+
+async checkPermission(){
+    const authStatus = await messaging().requestPermission();
+    const enabled =
+      authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+      authStatus === messaging.AuthorizationStatus.PROVISIONAL;
+    if (enabled) {
+      this.getFcmToken() 
+      console.log('Authorization status:', authStatus);
+    }
+}
+
+async getFcmToken(){
+    const fcmToken = await messaging().getToken();
+    if (fcmToken) {
+     console.log("Your Firebase Token is:", fcmToken);
+    } else {
+     console.log("Failed", "No token received");
+    }
+  }
```

Comment:
Do we need to show this error with proper error handling or that was just required for development?

Reply:
Han ji veer ji it is just required for development. it doesn't effect the user flow or give any problem to the user.